executor/kvm_gen_ppc64le.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
```

Comment:
s/2017/2021/

But can't we reuse [most of] kvm_gen.cc? It looks doable. We need something like:
```
#if AMD64
        PRINT(...);
        PRINT(...);
#elif PPC64LE
	PRINT(kvm_ppc64_mr);
	PRINT(kvm_ppc64_ld);
	PRINT(kvm_ppc64_recharge_dec);
#endif
```

Reply:
well, this requires defining AMD64 and PPC64LE, I could not find something well defined and widely supported quickly but I guess I can always add -D(AMD64|PPC64LE) in `go:generate bash -c ...`.

Reply:
Yes, I think it's the simplest. Please use the define names we use executor/defs.h (GOARCH_amd64).